docs/docs/ecommerce-tutorial/index.md

```diff
@@ -0,0 +1,542 @@
+---
+title: "Gatsby E-Commerce Tutorial"
+---
+
+# What this tutorial includes
+In this tutorial, you’ll learn how to use Gatsby to build the UI for a basic e-commerce site that can accept payments, with Stripe as the backend for processing payments. Benefits of using Gatsby for e-commerce sites include the following:
+* Security inherent in static sites
+* Blazing fast performance when your pages are converted from React into static files
+* Easy to host
+
+You can see the working demo hosted here: https://gatsby-eccomerce.netlify.com/
+
+
+# Background on Stripe
+
+Stripe is a payment processing service that allows you to securely collect and process payment information from your customers. To try out Stripe for yourself, go to [Stripe’s Quick Start Guide](https://stripe.com/docs/quickstart).
+
+There are alternatives to Stripe, like Square and Braintree, and their setup is very similar to Stripe.
+
+Stripe requires a server to process the information to make a charge, so you’ll need more than just static pages. Gatsby builds the pages for your front-end, but it won’t be able to handle the server logic that Stripe requires to process payments. That means you’ll need to set up a simple function that your Gatsby project can POST to in order to handle a payment.
+
+That function can be set up a number of different ways. To set up that function from scratch, you could:
+* Write your own simple server and deploy it somewhere, making it accessible via an endpoint
+* Write and deploy a hosted serverless function through a service like AWS Lambda or Google Cloud
+
+For this tutorial, you’ll use a serverless function (through AWS Lambda, though the steps in this tutorial will allow you to deploy to other cloud services in almost the same way). You also benefit from not having to keep a server running, because you only pay for the server time you use. Many companies that provide the ability to deploy serverless functions have a free tier that covers anywhere from several thousand to a million requests per month.
+
+[Insert a diagram showing a request flowing from Your static Gatsby site ===> Your serverless function ===> Stripe's servers and back again]
+
+
+# Setting up a Gatsby site
+
+Create a new Gatsby project by running the `gatsby new` command in the terminal and change directories into the new project you just started:
+
+```shell
+gatsby new ecommerce-gatsby-tutorial https://github.com/gatsbyjs/gatsby-starter-default
```

Comment:
Is the URL necessary? I thought gatsby new already used the default starter w/o needing a URL

Reply:
You're right the URL can be removed.